task/status.go

```diff
@@ -0,0 +1,39 @@
+package task
+
+type ActiveStatus string
+
+const (
+	StatusStarted ActiveStatus = "started"
```

Comment:
pending for me means "not started yet", so the order would be `pending -> started -> stopped`. But it's not necessary. Lets see how far we get with this :)